frontend/src/actions/logout.ts

```diff
@@ -12,6 +14,16 @@ export const signOutFunction = async () => {
     name: "access_token_secret",
     path: "/",
   });
+  cookies.delete({
+    name: "adonis-session",
+    path: "/",
+  });
+  cookies.delete({
+    name: "token",
+    path: "/",
+  });
```

Comment:
dlaczego to nie jest obsługiwane przez adonisa

Reply:
bo to jest po serverside, i to sie automatycznie nie ogarnia, tak samo jak musze recznie je ustawic i przekazywac

Reply:
ale jeszcze sie przyjrze

Reply:
no, ustawilem credentials include i nie przypisuje autmoayucznie cookiesow wiec musze recznie ustawiac i usuwac

Reply:
adonis powinien to ogarnąć, tylko w auth_controller jest tylko `response.clearCookie('token')`. Jest w to w `destroy()` w auth_controller.

Reply:
przykład z docsów adonisa: `import router from '@adonisjs/core/services/router'`

`router.delete('cart', async ({ request, response }) => { response.clearCookie('cart_items') })`